TYPO3.Flow/Classes/TYPO3/Flow/Core/Booting/Scripts.php

```diff
@@ -634,6 +634,23 @@ public static function executeCommand($commandIdentifier, array $settings, $outp
                 $exceptionMessage = implode(PHP_EOL, $output);
             } else {
                 $exceptionMessage = sprintf('Execution of subprocess failed with exit code %d without any further output. (Please check your PHP error log for possible Fatal errors)', $result);
+
+                // If the command is too long, it'll just produce /usr/bin/php: Argument list too long but this will be invisible
+                // If anything else goes wrong, it may as well not produce any $output, but might do so when run on an interactive
+                // shell. Thus we dump the command next to the exception dumps.
+                $exceptionMessage .= ' Try to run the command manually, to hopefully get some hint on the actual error.';
+
+                if (!file_exists(FLOW_PATH_DATA . 'Logs/Exceptions')) {
+                    mkdir(FLOW_PATH_DATA . 'Logs/Exceptions');
```

Comment:
We already use the Files Utility in this class. So we could use `Files::createDirectoryRecursively()`here. (Otherweise add a recursive flag)